docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md

```diff
@@ -44,15 +44,36 @@ Enter the command line interface on your machine, then paste and run the followi
 0. Find the `app_signed.hex` file under the `build/zephyr` directory in the project folder.
 
 ## Flash the binaries
-To program the board, use the nRF Connect Programmer with the `.hex` file generated or downloaded in a previous step.
-The whole process of [flashing Thingy:91 is described here.](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#update-nrf9160-application)
+To program the board, go through the whole process of **flashing Thingy:91** [described here.](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#update-nrf9160-application)
 
-After you have programmed and reset the board, go to the next step.
+After successful flashing, use the nRF Connect Programmer with the `.hex` file generated or downloaded in a previous step.
+Then reset the board and go to the next step.
+
+
+
+## Connect to the LwM2M Server
+
+To connect to Coiote IoT Device Management LwM2M Server, please register at https://eu.iot.avsystem.cloud/.
+
+To connect the board:
+
+0. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+0. In **Device Inventory**, click **Add device**.
+0. Select the **Connect your LwM2M device directly via the Management server** tile.
+    ![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+0. In the **Device credentials** step:
+     - In the **Endpoint name** field, enter your board endpoint name e.g. `test_device` (remember to set the same name in the [next section](#configure-the-client)).
+         ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+     - In the **Security mode** section, select the **Pre-Shared Key** mode:
+         - In the **Key identity** field, type `test_device`
```

Comment:
Nowhere does it say that the name "test_device" will be used by default as device name later in the tutorial, so the `In the **Key identity** field, type test_device` step sounds like each user should use `test_device` as Key identity which is not true. It will be the best to write that `It is recommended to use endpoint name as key identity`.